SConstruct

```diff
@@ -774,7 +774,7 @@ else:
     # We apply it to CCFLAGS (both C and C++ code) in case it impacts C features.
     env.Prepend(CCFLAGS=["/std:c++17"])
 
-# Disable exception handling. Godot doesn't use exceptions anywhere, and this
+# Disable exception handling. Blazium doesn't use exceptions anywhere, and this
```

Comment:
Please revert changes to comments, it makes merge conflicts much more likely in the future.

Reply:
will revert the non-required changes in comments in the squashed commit